### DISCUSSION TITLE
Add separate combinator between each rule

### DISCUSSION THREAD

**Discussion by mistrya** [react-querybuilder/react-querybuilder#759] [Q&A]
Hello 
How can we add separate combinator between each rule  to construct a query like below - 
FirstName = 'John' AND LastName = 'Smith' OR LastName = 'Janes'

**Comment by jakeboone02** [comment]
The feature you're referring to is called "independent combinators" and is documented here:

https://react-querybuilder.js.org/docs/components/querybuilder#independent-combinators
